### PR TITLE
Revert "dataset .dvc files"

### PR DESCRIPTION
**Pull request by SoyGema**
Reverts iterative/dataset-registry#33
Last commit wasn´t included

### CODE REVIEW

**Comment by SoyGema** [comment]
- [X] Add README.md 
- [X] Remove unnecessary .dvc file

**Comment by SoyGema** [comment]
- [X] Remove .gitignore root folder.

File: workshop/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+## Dataset description
```
Comment (shcheklein) [inline workshop/README.md:1]: I think we need primarily some info on where this dataset is being used, if it's from Kaggle and structure is described there - no need to repeat much tbh.

Also, please try to wrap it at 80 like we do in the other projects - docs, blog, etc.

**Review by shcheklein — CHANGES_REQUESTED** [review]
Thanks @SoyGema ! A small stylistic comment.

**Comment by shcheklein** [comment]
> * [x]  Remove .gitignore root folder.

I don't see any changes to it? Expected?

File: workshop/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+The dataset contains information about RSOs - artificial objects that are in 
+orbit around earth used to predict orbit trajectories. 
+
+Each row represents an orbit observation of a satellite. There are 600 
+satellites that have multiple observations.
+
+### Acknowledgments
+
+Dataset published in [Kaggle](https://www.kaggle.com/datasets/idawoodjee/predict-the-positions-and-speeds-of-600-satellites) and has been collected by
```
Comment (shcheklein) [inline workshop/README.md:14]: same here, please format

there are plugins for VS Code that can do wrap automatically - e.g. https://marketplace.visualstudio.com/items?itemName=stkb.rewrap

File: workshop/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+
```
Comment (shcheklein) [inline workshop/README.md:1]: @SoyGema extra space are not needed

**Review by shcheklein — APPROVED** [review]
Thanks @SoyGema ! :)

**Pull request merged**